csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html

```diff
@@ -0,0 +1,215 @@
+{% extends "csss/header.html" %}
```

Comment:
also, please use the CSS of the https://dev.sfucsss.org/PR-420/events/mm/2021 for this page as well.

the website already looks like shit, let's try and not make it worse and maintain some consistency on these special pages 😅

Reply:
This was consistent with the Fall Hacks Submission page.